app/helpers/tax_return_status_helper.rb

```diff
@@ -1,12 +1,17 @@
 module TaxReturnStatusHelper
-  def grouped_status_options_for_select
+  def grouped_status_options_for_select(unwanted_statuses: nil)
     TaxReturnStateMachine.available_states_for(role_type: current_user.role_type).map do |stage, statuses|
       translated_stage = TaxReturnStatusHelper.stage_translation(stage)
-      translated_statuses = statuses.map { |status| [TaxReturnStatusHelper.status_translation(status), status.to_s] }
+      filtered_statuses = unwanted_statuses ? statuses.reject { |status| unwanted_statuses.include?(status.to_s) } : statuses
```

Comment:
i'd set the default `unwanted_statuses` to `[]` and then just do
```suggestion
      filtered_statuses = unwanted_statuses.present? ? statuses.reject { |status| unwanted_statuses.include?(status.to_s) } : statuses
```

Reply:
can also do:
```suggestion
      filtered_statuses = statuses.reject { |status| unwanted_statuses.include?(status.to_s) }
```

but we'd unnecessarily try to reject when that `unwanted_statuses` is empty so I guess it's nice to have that check